src/screens/AddSiteFormScreen.js

```diff
@@ -1,76 +1,26 @@
 import React, {Component} from 'react';
-import {Alert, Picker, Platform, ScrollView, View, Image} from 'react-native';
+import {Platform} from 'react-native';
 import {connect} from 'react-redux';
-import {Icon, Text, CheckBox, Input} from 'react-native-elements';
+import {Icon} from 'react-native-elements';
 
-import _ from 'lodash';
-import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view';
+import {NavbarButton} from '../components/common/';
+import SiteInfoInputForm from '../components/SiteInfoInputForm';
```

Comment:
Most of this code came out and was refactored into the `SiteInfoInputForm` component